README.rst

```diff
@@ -43,6 +43,8 @@ to 2.7 and don't require anything extra.
 
 For ``openssl``, step 5 will be more complicated :)
 
+For ``libffi``, before step 5 each include/<arch>/fficonfig.h and 
+include/<arch>/ffi.h must be generated by running configure_headers.bat.
```

Comment:
Once the file is moved, this line will need updating.

Reply:
The line should be updated now